quantus/metrics/faithfulness/faithfulness_correlation.py

```diff
@@ -387,7 +328,59 @@ def evaluate_batch(
         scores_batch:
             The evaluation results.
         """
-        return [
-            self.evaluate_instance(model=model, x=x, y=y, a=a)
-            for x, y, a in zip(x_batch, y_batch, a_batch)
-        ]
+        # Prepare shapes. Expand a_batch if not the same shape
+        if x_batch.shape != a_batch.shape:
+            a_batch = np.broadcast_to(a_batch, x_batch.shape)
+
+        # Flatten the attributions.
+        batch_size = a_batch.shape[0]
+        a_batch = a_batch.reshape(batch_size, -1)
+        n_features = a_batch.shape[-1]
+
+        # Predict on input.
+        x_input = model.shape_input(
+            x_batch, x_batch.shape, channel_first=True, batched=True
```

Comment:
afaik `channel_first` is a models parameter, so we should not hardcode it. @annahedstroem could you please help us on that one 🙃

Reply:
This was hardcoded in the original implementation as well. Is that a bug?